docs/content/release_notes/csi_rn_limitations.md

```diff
@@ -29,4 +29,11 @@ The following limitations apply when expanding volumes with the IBM block storag
 
 -   When using the CSI driver with IBM Spectrum Virtualize Family and IBM DS8000 Family products, during size expansion of a PersistentVolumeClaim (PVC), the size remains until all snapshots of the specific PVC are deleted.
 -   When expanding a PVC while not in use by a pod, the volume size immediately increases on the storage side. PVC size only increases, however, after a pod begins to use the PVC.
--   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning, and so on) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
\ No newline at end of file
+-   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
+
+## Volume replication limitations
+
+When a role switch is conducted, this is not reflected within the other orchestration platform replication objects.
+
+**Important:** When implementing volume replication, be sure to use the Retain reclaim policy for the PersistentVolume and using the latest import procedure (version 1.7.0 or later) (see [Importing an existing volume](../configuration/csi_ug_config_advanced_importvol.md) in the User Guide).
```

Comment:
as I [wrote](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r716161175):
1. I'm not sure it's clear that "use the Retain reclaim policy" means "delete the PV using Retain reclaim policy".
2. IMO it is also not completely clear that the relevant PV is of the replication dataSource.

Also I don't recall us ever using the term `implementing` for user configurations, I think it's confusing, since we are the ones that implemented it. I suggest using `using`/`configuring` instead

Reply:
DONE

Reply:
continuing [here](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r718312918)